src/main/java/cheese/command/Command.java

```diff
@@ -0,0 +1,36 @@
+package cheese.command;
+
+import cheese.CheeseException;
+import cheese.Storage;
+import cheese.TaskList;
+import cheese.Ui;
+
+/**
+ * Encapsulates small parts of logic for the bot
+ */
+public class Command {
+    private final boolean exitChat;
```

Comment:
To follow the coding standard, perhaps it would be better to rename this boolean to `isExitChat` or `isExitCommand` instead.